.github/workflows/compiler_typescript.yml

```diff
@@ -89,4 +72,4 @@ jobs:
           path: "**/node_modules"
           key: ${{ runner.arch }}-${{ runner.os }}-modules-${{ hashFiles('compiler/**/yarn.lock') }}
       - run: yarn install --frozen-lockfile
-      - run: yarn workspace ${{ matrix.workspace_name }} test
+      - run: yarn workspaces run test
```

Comment:
something annoying about yarn workspaces is that this will run the `test` command on every package including the playground which is why I originally added that one liner to only find packages in that `packages` directory